src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -1917,10 +1917,7 @@ void GameObject::Use(Unit* user)
                 if (level < info->meetingstone.minLevel)
                     return;
 
-                if (info->entry == 194097)
-                    spellId = 61994;                            // Ritual of Summoning
```

Comment:
Why

Reply:
3.4.3 sniffs show no casting of 61994, but 23598 does show up.

Reply:
Just to make sure you checked right. One is for summon stone one is for lock portal

Reply:
Yes, I've been staring at sniffs for both of them for a couple days at this point, I am quite confident in what I've done in this PR (assuming the sniffs aren't missing information).